src/pages/speakers/speakers-tips/index.astro

```diff
@@ -7,27 +7,27 @@ import {Image} from "astro:assets"
 <Layout title="JavaZone 2024 - Speakers - Submission tips">
     <div class="grid md:inline-block">
         <h1 class="text-[5.5vw] mx-auto md:text-[4vw]">Submission tips for speakers</h1>
-        <div class="text-[4vw] mx-auto md:text-[2.5vw]">
+        <div class="text-[4vw] mx-auto md:text-[2vw]">
             <ButtonLink href="https://talks.javazone.no/" target="_blank">Submit your talk!</ButtonLink>
         </div>
-        <h2 class="text-[4vw] md:text-[2.5vw]">Increase the chance of your talk being accepted</h2>
-        <p class="text-[3.5vw]  md:text-[2.5vw]">
+        <h2 class="text-[4.5vw] md:text-[2vw]">Increase the chance of your talk being accepted</h2>
```

Comment:
er det sånn at md:text er fontstørrelse på mindre skjermer? Og text på større skjermer? Hvis ja, hvorfor øker du størrelse på teksten som allerede var for stor fra 4 til 4.5?